Docker/dbadmin.Dockerfile

```diff
@@ -10,9 +10,9 @@ ENV POSTGRES_USER=${POSTGRES_USER}
 ENV POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
 ENV POSTGRES_DB=postgres
 
-COPY Docker/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
+COPY Settings/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
 COPY Artifacts/PgSql/Structure/Admin/ /tmp/AdminApiScripts/PgSql
```

Comment:
@lchavarriaga 
Path needs to be updated to match folder structure on docker context.